common.js

```diff
@@ -3232,6 +3232,146 @@ const allSoloCars = {
       'all': ['bsp'],
     },
   },
+  'Lotus': {
+    '7 & 7A': {
+      'all': ['cs', 'ssp']
+    },
+    '7 & 7A (948, 997, & 1098 cc)': {
+      'all': ['dp']
+    },
+    'Cortina': {
+      'all': ['csp', 'hs']
+    },
+    'Eclat': {
+      'all': ['cs', 'ssp']
+    },
+    'Elan': {
```

Comment:
Combine all Elan models into this one and remove the others. The classes should be `cs, ssp, dp`. Thanks!